gr-filter/include/gnuradio/filter/fft_filter.h

```diff
@@ -26,6 +26,7 @@
 #include <gnuradio/fft/fft.h>
 #include <gnuradio/filter/api.h>
 #include <gnuradio/gr_complex.h>
+#include <volk/volk_alloc.hh>
```

Comment:
How did CI not catch this? There's an `h` to much...

Reply:
Is it? It's not just the indicator saying it's a C++-only header?

Reply:
Ah, I see you reverted that commit now.